app/config/services.yaml

```diff
@@ -26,3 +26,9 @@ services:
 
     # add more service definitions when explicit configuration is needed
     # please note that last definitions always *replace* previous ones
+
+    scheb_two_factor.trusted_jwt_encoder.configuration.key:
```

Comment:
I didn't want to change `scheb_two_factor.trusted_jwt_encoder.configuration.algorithm` instead to keep compatibility with older `lcobucci/jwt` versions :+1: